.env.example

```diff
@@ -27,6 +27,9 @@ RECAPTCHA_SECRET_KEY=6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe
 
 REDIS_HOST=redis
 
+SLACK_API_TOKEN=xoxb-***
+SLACK_CHANNEL=matchbot
```

Comment:
@bdsl My thinking was mostly that it might be nice to vary it by environment or purpose of message in the future, especially if adding more types of notification

Reply:
Personally I'd prefer go with YAGNI and only put in the variability we actually want now, not stuff we might want to vary in future. To me it actually seems easier to make the channel vary by purpose in future if the PHP code is fully responsible for knowing what channel to write to.

Reply:
Not a blocker anyway.

Reply:
Mm interesting. I think I tend to veer maybe too much towards adding more vars up front because coordinating deployments later is more fiddly if we are also changing env vars in tandem. Whereas deleting them is less error prone.